.circleci/config.yml

```diff
@@ -485,8 +485,8 @@ workflows:
             mkdir -p test-reports/itest
             cd tests/integration
             python3 -m records.multi_db.test_records_table2table
-          requires:
-            - redshift-itest
+#          requires:  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker